website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another Windows Subsystem for Linux (WSL) distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, [Podman Desktop creates a Windows Subsystem for Linux (WSL) virtual machine: the Podman Machine](/docs/podman/creating-a-podman-machine.md).
+It also configures the Windows Podman client to communicate with the Podman Machine.
+However, it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the most common context to walk you through the steps to configure your WSL distribution:
+
+- Ubuntu distribution of Linux.
+- Default Podman Machine.
+
+In foldable details, you can find alternative steps for least common contexts:
+
+- Custom WSL distribution.
+- Custom Podman Machine.
+
+## Configuring your WSL distribution
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. To communicate with the remote Podman Machine, you need a Podman client.
+
+   To benefit from the latest features, such as `podman kube` subcommands, and to enable the remote mode by default, consider installing the `podman-remote` binary rather than the `podman` package from the distribution.
+
+   Download the `podman-remote` binary from the [Podman releases page](https://github.com/containers/podman/releases/latest).
```

Comment:
The experience to complete this step when running on WSL/Ubuntu leaves a lot to be desired. First going to the link in the latest release, it does not show podman-remote as an asset, you have to scroll down and open the fold before you even find it (I had assumed it was a bad link). Then there are multiple binaries with no explanation on which to use nor how to install. I had seen multiple comments in forums stating that `podman-remote-static` was not compatible with Podman Desktop, so I thought it would be simpler to install the full Podman. However this is a waste of time on Ubuntu because 3.x does not support the necessary commands. I spent ages trying to find a way to install Podman 4.x on Ubuntu, but I wish someone had warned me just not to bother. So finally I went back to explore podman-remote. I recommend the following changes to make this is smoother experience:

The Podman client is available with a full `podman ` installation or with the `podman-remote` version 4.x and higher. On Ubuntu it is generally easier to install `podman-remote`

Check for the latest release which includes the `podman-remote` binary from the [Podman releases page](https://github.com/containers/podman/releases/latest)
Download and unpack the binary:
`wget https://github.com/containers/podman/releases/download/v4.9.1/podman-remote-static-linux_amd64.tar.gz`
`sudo tar -C /usr/local -xzf podman-remote-static-linux_amd64.tar.gz`
Make this executable as `podman` with the following additions to `.basrc`:
export PATH="$PATH:/usr/local/bin"